docs/src/main/asciidoc/config-yaml.adoc

```diff
@@ -209,3 +209,45 @@ quarkus:
 
 YAML `null` keys are not included in the assembly of the configuration property name, allowing them to be used
 in any level for disambiguating configuration keys.
+
+Although Quarkus primarily uses `.properties` file extension for configuration, the snakeyaml library, which is used for parsing YAML in Quarkus, can also parse JSON structures. This means you can use YAML files with JSON content inside.
```

Comment:
There is no need to reference snakeyaml, since this is a hidden implementation detail. My recommendation is to just state that a JSON structure can be used in a YAML file and use the example.

Thanks!

Reply:
Hi @radcortez 
the above is fixed.